include/dfrn.php

```diff
@@ -1833,8 +1836,11 @@ private static function process_relocation($xpath, $relocation, $importer) {
 						$n, dbesc($f[1]),
 						$n, dbesc($f[0]),
 						intval($importer["importer_uid"]));
-					if ($x === false)
+
+					/// @TODO decide to use dbm::is_result() here
```

Comment:
Unworkable `@TODO`, do it or don't, but since it's an UPDATE result, it's not crucial.